app/models/spree/calculator/weight/package_builder.rb

```diff
@@ -15,11 +15,26 @@ def process(package)
         packages(package)
       end
 
+      # Configuration 
+      def units
```

Comment:
Are any of these going to be part of the public interface of the PackageBuilder? If not, i'd prefer if they were `private`.